src/elements/element.line.js

```diff
@@ -21,10 +21,11 @@ module.exports = function(Chart) {
 		lineToNextPoint: function(previousPoint, point, nextPoint, skipHandler, previousSkipHandler) {
 			var me = this;
 			var ctx = me._chart.ctx;
+			var spanGaps = me._model ? me._model.spanGaps : false;
```

Comment:
@etimberg: is it really `_model` when drawing? I thought we should use `_view` instead, resulting of the element pivot().